### DISCUSSION TITLE
Pinter Killed. Err: MINTEMP.

### DISCUSSION THREAD

**Discussion by zeblis** [mriscoc/Ender3V2S1#821] [Q&A]
Is there any firmware changes that could be causing this MINTEMP error when I try to preheat the printer?  Happens both when I try to heat the printer from the screen, as well as octoprint.  Wieird thing is, I can eventually get the printer to heat and print if I first heat the bed, then heat the hot end(This still took a few tries).  However this stopped working after a while.  I then discovered that I could print if I first turned on parts cooling, then heat the bed, and then heat the hot end(This is also hit or miss).  I've already changed the thermistor.  I've also scanned the configuration.h and configuration_adv.h for any potential settings I may have missed.  The printer was running perfect with Jyers 2.0.1 before, but I extended my Ender 3v2 and struggled to compile it for a larger bed. Also there were many deprecated code issues.  I tried Marlin 2.1.2 from github, but the menu icons broke, even with the provided matching DWIN_Set.  MRISCOC had all the matching DWIN_set and features, but this MINTEMP Issue I'm experiencing now.

I only things I was able to find online involved faulty thermistor or ambient temps being too low.  Both thermistor properly show ambient temps.  The printer was previously printing fine for months, and returning the the previous Jyers 2.0.1 fork, and a my own compile of Marlin 2.1.2 heats from screen menus and octoprint controls without issue.  I copied my settings from my working Jyers 2.0.1 fork.

This is my first time compiling Marlin and I would really appreciate any insights if you've dealt with this same issue, or if I may have missed any settings.  Thanks ahead of time.

**Comment by mriscoc** [comment]
Try with Marlin but in configuration.h add:
```cpp
#define USE_STOCK_DWIN_SET
```
also go back to PID:
```cpp
#define PIDTEMP          // See the PID Tuning Guide at https://reprap.org/wiki/PID_Tuning
//#define MPCTEMP        // ** EXPERIMENTAL **
```

**Reply by zeblis** [reply]
Thank you for the response.  Here is what I tried, and what the results are now.
First I check the PID Settings.  #define PIDTEMP was currently active, so I tried PID Tuning again just in case.  Same issue.

I then tried switching back to Marlin 2.1.2 with #define USE_STOCK_DWIN_SET, and updated LCD with the Marlin 2.1.2 DWIN_SET.  It booted, but the screen is now blank.

I then switched back to the MRiscoC with matching DWIN_SET.  Screen icons are back, and the problem is back.  However, on the first boot after updating screen, preheat PLA from screen worked like normal.  After rebooting again, preheat from PLA fails again.  On the third try, I accidentally click preheat ABS, and it heats normally.  Switching to preheat PLA worked normally.  I reboot it again, and the error comes back.  

I tested several times again, and preheating ABS, then switching to preheat PLA, works.  At least I have fewer clicks to get started printing.

As soon as I try this trick on Octoprint, the error comes back, and this trick no longer works from the screen.  Turning it off for a few minutes, I got the trick to work from the screen again.  I'll post a video of the intermittent issue that I recorded right before trying to preheat from octoprint.

Thanks again!

**Reply by zeblis** [reply]
Here is a link to the video:  Thanks again!
https://youtu.be/LKIZUFGe14Y

**Reply by mriscoc** [reply]
> I then tried switching back to Marlin 2.1.2 with #define USE_STOCK_DWIN_SET, and updated LCD with the Marlin 2.1.2 DWIN_SET. It booted, but the screen is now blank.

You must use the stock DWIN icon set if use `#define USE_STOCK_DWIN_SET` if you change to Marlin icon set, that "#define" must be disabled.

If you use Octoprint, be sure that +VUSB is disabled.

**Reply by zeblis** [reply]
Yes, I have the DWIN SET from the Configurations for the E3V2 screen with #define USE_STOCK_DWIN_SET.  
And yes, the +V on the USB is taped off to  disable it.

**Comment by mriscoc** [comment]
Why don't you try the precompiled 422 version?

**Reply by zeblis** [reply]
I'll have to try the the precompiled 422.  I just found the ender extender github.  I started down the self-compile road, as I didn't think there was a pre-compiled version that included the bed size and BLTouch.  We'll have to see how that works.  
Thanks again for our responsiveness!

**Reply by gsuresh2u** [reply]
I am also using ender extender with skr mini e3 v3.0
If you want any help for compiling your own, I will help you
Post your Config files

**Reply by mriscoc** [reply]
For the extender use: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by zeblis** [reply]
> I am also using ender extender with skr mini e3 v3.0 If you want any help for compiling your own, I will help you Post your Config files

[mriscoc Ender3V2S1-20230312.zip](https://github.com/mriscoc/Ender3V2S1/files/11074522/mriscoc.Ender3V2S1-20230312.zip)

I appreciate your offer, I'm running the 422 board E3V2 with 512k.  I'm also using the stock creality icon set.  Feel free to fix anything I may have missed.

300x300x400 with BL Touch. let me know if you have any questions for me.  Thanks again.  I'm happy to try your compilation.

**Reply by zeblis** [reply]
> For the extender use: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

I'm tying this Ender3V2-422-BLTUBL-MPC-20230312.bin  version with https://github.com/mriscoc/Ender3V2S1/tree/Ender3V2S1-Released/display%20assets/stock. DWIN.

The display is just blank.  I must be missing something.

**Reply by mriscoc** [reply]
That version works fine. Try renaming the firmware file.

**Reply by zeblis** [reply]
Yup, I renamed it firmware-20230312.bin.  I tried renaming it again to just firmware.bin without the numbers and it flashed properly.  

I started the calibration Guide, physical prep, min and max.  It's currently pre-heating normally.  I am grateful.  Thank you again!
